### PR TITLE
rename ifc model to simple name to avoid issues in the conversion pipeline

### PR DESCRIPTION
**Pull request by machisuji**
e.g. IfcConvert doesn't seem to be able to handle umlauts in filenames

WP [#32806](https://community.openproject.com/projects/bim/work_packages/32806/activity)

### CODE REVIEW

File: modules/bim/app/services/bim/ifc_models/view_converter_service.rb [hunk 1]
```diff
@@ -68,20 +68,38 @@ def call
...
 
-        # Step 2: Collada2GLTF
-        gltf_file = convert_to_gltf(collada_file, dir)
+        FileUtils.cp ifc_path.to_s, tmp_ifc_path.to_s
 
-        # Step 3: Convert to XKT
-        xkt_file = convert_to_xkt(gltf_file, dir)
-        ifc_model.xkt_attachment = File.new xkt_file
+        tmp_ifc_path.to_s
+          .then { |ifc_file| convert_to_collada ifc_file, dir } # Step 1: IfcConvert
```
Comment (wielinde) [inline modules/bim/app/services/bim/ifc_models/view_converter_service.rb]: naming: `ifc_file` is a pathname not a file

File: modules/bim/app/services/bim/ifc_models/view_converter_service.rb [hunk 2]
```diff
@@ -68,20 +68,38 @@ def call
...
-        # Step 2: Collada2GLTF
-        gltf_file = convert_to_gltf(collada_file, dir)
+        FileUtils.symlink ifc_path.to_s, tmp_ifc_path.to_s
 
-        # Step 3: Convert to XKT
-        xkt_file = convert_to_xkt(gltf_file, dir)
-        ifc_model.xkt_attachment = File.new xkt_file
+        tmp_ifc_path.to_s
+          .then { |ifc_file| convert_to_collada ifc_file, dir } # Step 1: IfcConvert
+          .then { |collada_file| convert_to_gltf collada_file, dir } # Step 2: Collada2GLTF
```
Comment (wielinde) [inline modules/bim/app/services/bim/ifc_models/view_converter_service.rb]: Didn't know the `.then` syntax. Very nice.

**Review by wielinde — COMMENTED** [review]
Hey @machisuji , thank you for the PR. I haven't tested it yet, but it looks as if it does what it should. I like the `.then` syntax. It's beautiful.

However, I have some difficulties with the other methods. I find them hard to read due to the naming.

**Comment by machisuji** [comment]
@wielinde I should have addressed your concerns.

**Review by wielinde — APPROVED** [review]
Thank you @machisuji! It works nicely and is super readable. And a special thanks for providing the spec.

**Pull request merged**